docs/02.API-REFERENCE.md

```diff
@@ -22,6 +22,21 @@ Possible types of an error:
  - JERRY_ERROR_TYPE - type error
  - JERRY_ERROR_URI - URI error
 
+## jerry_feature_t
+
+Possible types of a feature:
+
+ - JERRY_FEATURE_ERROR_MESSAGES - error messages
+ - JERRY_FEATURE_VALGRIND - valgrind support
+ - JERRY_FEATURE_VALGRIND_FREYA - valgrind-freya support
+ - JERRY_FEATURE_CPOINTER_32_BIT - 32 bit compressed pointers
+ - JERRY_FEATURE_MEM_STRESS_TEST - mem-stress test
+ - JERRY_FEATURE_MEM_STATS - memory statistics
+ - JERRY_FEATURE_PARSER_DUMP - parser byte-code dumps
+ - JERRY_FEATURE_REGEXP_DUMP - regexp byte-code dumps
+ - JERRY_FEATURE_SNAPSHOT_SAVE - saving snapshot files
+ - JERRY_FEATURE_SNAPSHOT_EXEC - executing snapshot files
```

Comment:
Thanks the reviews. I removed the mentioned features from the list, and chose an intermediate solution from your suggestions. Please check it.